src/main/java/uk/ac/ox/oxfish/biology/growers/CommonLogisticGrower.java

```diff
@@ -51,46 +51,28 @@ public void step(SimState simState) {
 
     protected void grow(FishState model, List<BiomassLocalBiology> biologies) {
 
-        grow(model,biologies,biologies);
-    }
-
-
-    protected void grow(FishState model, List<BiomassLocalBiology> allBiologies,
-                        List<BiomassLocalBiology> nonFadHabitats) {
-
         double current = 0;
         double capacity = 0;
         //for each place
-        for(VariableBiomassBasedBiology biology : allBiologies)
-        {
+        for (VariableBiomassBasedBiology biology : biologies) {
```

Comment:
glad to see this changing back, but why is it not an issue anymore?

Reply:
It's still is, but we now have `FadAwareLogisticGrower`. We had a regression with taking the biomass under FADs into account. I could have fixed it there, but it was simpler to handle it in a single class instead of the previous `CommonLogisticGrower` /  `SchaeferLogisticGrower` / `FadAwareCommonLogisticGrower` hierarchy.